API/api/controllers/event_controller.py

```
@@ -3,11 +3,40 @@
+
+    event_date_raw = request.data.get("event_date")
+    name = request.data.get("name")  # Optionnel, peut être utilisé pour mettre à jour le nom en même temps
+    event_date = None
+
+    if not event_date_raw:
+        raise ValidationError({"detail": "event_date est requis (YYYY-MM-DD)."})
+
+    try:
+        event_date = datetime.fromisoformat(event_date_raw).date()
+    except ValueError:
+        raise ValidationError({"detail": "event_date invalide (format attendu: YYYY-MM-DD)."})
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)